Android/Io.OpenCensus/source/Xamarin.Io.OpenCensus.OpenCensusContribGrpcMetrics/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using Android.App;
+
+// Information about this assembly is defined by the following attributes. 
+// Change them to the values specific to your project.
+
+[assembly: AssemblyTitle("Xamarin.Protobuf.Lite.")]
+[assembly: AssemblyDescription("Xamarin bindings for Protobuf Lite package")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Microsoft Corporation")]
+[assembly: AssemblyProduct("")]
+[assembly: AssemblyCopyright("Microsoft Corporation 2018")]
```

Comment:
I think this needs fixing too, should be `© Microsoft Corporation. All rights reserved.`

Reply:
done